Recategorize items in the Table of Contents (#68)
UpliftGames/moonwave: website/docs/Configuration.md

classes = ["Class7", "ClassAte", "Class9"]
```

### Table of Contents (TOC)

> You can also customize the categories in each LuaClass page's table of contents (TOC). To add a category, first add it as a `tag` field within a doc comment for a class's function.

Reviewer: Here we are using `LuaClass`, which is an internal name, to refer to something the user thus far only knows as an "API page". We should use this terminology instead.

```suggestion
You can customize categories in the Table of Contents of each API page. To create a category, first tag all items that should be included in the category with the `@tag` tag.
```

```lua
--[=[
	This is a very fancy function that adds a couple numbers.

	@param a number -- The first number you want to add
	@param b number -- The second number you wanna add
	@return number -- Returns the sum of `a` and `b`

>     @tag utility

Reviewer: Mixed indentation - run formatter

]=]
function MyFirstClass:taggedFunction(a, b)
	return a + b
end
```

> Then to break-out the new table of contents category, specify it under the `apiCategories` option in your `moonwave.toml` file.

Reviewer: ```suggestion
Then, specify those tags under the `apiCategories` option in your `moonwave.toml` file.
```
